pkg/controller/schedule_controller_test.go

```diff
@@ -239,27 +238,13 @@ func TestProcessSchedule(t *testing.T) {
 			if test.expectedLastBackup != "" {
 				require.True(t, len(actions) > index, "len(actions) is too small")
 
-				patchAction, ok := actions[index].(core.PatchAction)
-				require.True(t, ok, "action is not a PatchAction")
-
-				patch := make(map[string]interface{})
-				require.NoError(t, json.Unmarshal(patchAction.GetPatch(), &patch), "cannot unmarshal patch")
-
-				assert.Equal(t, 1, len(patch), "patch has wrong number of keys")
-
-				lastBackup, _ := collections.GetValue(patch, "status.lastBackup")
-				fmt.Println(lastBackup)
-
-				assert.True(
-					t,
-					collections.HasKeyAndVal(patch, "status.lastBackup", parseTime(test.expectedLastBackup).UTC().Format(time.RFC3339)),
-					"patch's status.lastBackup does not match",
-				)
-
-				res, _ := collections.GetMap(patch, "status")
-				assert.Equal(t, 1, len(res), "patch's status has the wrong number of keys")
+				expected := Patch{
+					Status: PatchStatus{
+						LastBackup: parseTime(test.expectedLastBackup),
+					},
+				}
 
-				index++
+				arktest.ValidatePatch(t, actions[index], expected, decode)
```

Comment:
We still need to `index++`, right?

Reply:
We don't need this one since this is the last action to be validated (this isn't in a loop, there are just 3 static action checks). Was probably there before just from copy-pasting each block.